### PR TITLE
Invoice fix

### PR DESCRIPTION
**Pull request by almathew**

### CODE REVIEW

File: postajob/views.py [hunk 1]
```diff
@@ -51,13 +51,15 @@ def jobs_overview(request):
 @company_has_access(None)
 def view_job(request, purchased_product, pk, admin):
     company = get_company_or_404(request)
-    product = PurchasedProduct.objects.get(pk=purchased_product)
-    if not product.owner == company:
+    purchased_product = PurchasedProduct.objects.get(pk=purchased_product)
+    if admin and not purchased_product.product.owner == company:
```
Comment (tdphillips) [inline postajob/views.py]: Would xor be too obscure?

`if bool(admin) ^ purchased_product.product.owner == company`

Comment (aspidites) [inline postajob/views.py]: Probably. Isn't `bool()` implied by `if`, or does the XOR operator mandate its use?

Comment (almathew) [inline postajob/views.py]: I made it a 2 line if statement specifically for readability already...

Comment (tdphillips) [inline postajob/views.py]: If admin were an instance of some model like I was originally thinking, `bool()` would be necessary. I think it's actually a bool already, however, so it's unnecessary.

**Pull request merged**